pkg/strategy/xalign/metrics.go

```diff
@@ -0,0 +1,54 @@
+package xalign
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var quantityDeltaMetrics = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "xalign_balance_quantity_delta",
+		Help: "The balance delta of the currency",
+	},
+	[]string{"currency", "side"},
+)
+
+var amountDeltaMetrics = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "xalign_balance_amount_delta",
+		Help: "The balance delta of the quote amount of the currency",
+	},
+	[]string{"currency", "side"},
+)
+
+func (s *Strategy) updateMetrics(currency string, side string, quantityDelta float64, amountDelta float64) {
+	var quantityGauge, amountGauge prometheus.Gauge
+	key := currency + "_" + side
```

Comment:
can we define a struct for the key?

type metricsKey struct { currency string; side string ; ... }